src/main/kotlin/app/Configurator.kt

```diff
@@ -0,0 +1,199 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+
+package app
+
+import app.utils.Options
+import app.utils.PasswordValidator
+import app.utils.UsernameValidator
+import org.apache.commons.configuration.ConfigurationException
```

Comment:
[3.3.3 Ordering and spacing](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements) says that import statements have to be sorted in ascii sort order.